libs/dh/market-participant/actors/feature-actors/src/lib/create/dh-actors-create-actor-modal.component.ts

```diff
@@ -138,6 +140,7 @@ export class DhActorsCreateActorModalComponent {
     glnOrEicNumber: ['', [Validators.required, dhEicOrGlnValidator]],
     name: [''],
     marketrole: ['', Validators.required],
+    gridArea: ['', Validators.required],
```

Comment:
question: this control is required but the dropdown is only shown when the selected market role is `GridAccessProvider`. Can the user still go to the next step if the selected market role is something different than `GridAccessProvider`?